web/packages/test/scripts/start-relayer.sh

```diff
@@ -79,6 +79,19 @@ start_relayer() {
             sleep 20
         done
     ) &
+    # Launch beefy on demand relay
```

Comment:
In the local env, the session length is like only 10 blocks, which means the on-demand relay isn't necessary and could conflict with the main beefy relayer.

What behavior did you observe in the local env?

Maybe we could add a check in the on-demand relay, where if it sees that the session length is less than 10 minutes it does nothing?

Reply:
> the session length is like only 10 blocks

Actually, it's 20 blocks (2 mins), so I can test it within a reasonable time window. Since it's a local setup, I assume it's not a big deal even if it might cause conflicts, and I'd prefer to add it.

Reply:
It is problematic in a local env if it causes conflicts and undefined behavior. We want local env to be repeatable and consistent.